src/shogun2-core/shogun2-dao/src/main/java/de/terrestris/shogun2/dao/AbstractTokenDao.java

```diff
@@ -0,0 +1,27 @@
+package de.terrestris.shogun2.dao;
+
+import org.springframework.stereotype.Repository;
+
+import de.terrestris.shogun2.model.token.Token;
+
+/**
+ * As the {@link Token} class is abstract, this class will also be abstract.
+ * There will also be NO {@link Repository} annotation here.
```

Comment:
Haha. You're right. The import is only there because `Repository` is mentioned in the JavaDoc. If you want the `Repository` being linked in the JavaDoc, we should not change this. ;-)